distributed/client.py

```diff
@@ -2280,6 +2281,7 @@ def publish_dataset(self, *args, **kwargs):
         ----------
         args : list of objects to publish as name
         name : optional name of the dataset to publish
+        override : if true, override any already present dataset with the same name
```

Comment:
If false, will raise when the key already exists.
These argument descriptions could use a better style, like
```
override : bool (optional, default False)
    if true, override ...
```